spec/controllers/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+describe V1::UsersController, type: :controller do
+
+  describe 'GET #show' do
+    before(:each) do
+      @user = FactoryGirl.create :user
```

Comment:
Another comment about creating objects in tests - I prefer to use the `let` and `let!` syntax rather than putting them in `before` blocks. Here's this same spec setup using `let`:
```
describe 'GET #show' do
  let(:user) { create(:user) }

  before(:each) do
    get :show, id: user.id, format: :json
  end

  it 'returns user information' do
    user_response = JSON.parse(response.body, symbolize_names: true)
    expect(user_response[:email]).to eql user.email
    expect(response.status).to eq(200) 
  end
...
```
For more info on this, check out: http://betterspecs.org/#let